src/helpers/files.ts

```diff
@@ -58,6 +58,15 @@ export const matchesRewrite = (file: string, rewrites: Rewrites): boolean => {
   return matchesRedirect(file, rewrites.beforeFiles)
 }
 
+export const getMiddleware = async (publish: string): Promise<Array<string>> => {
+  const manifestPath = join(publish, 'server', 'middleware-manifest.json')
+  if (existsSync(manifestPath)) {
+    const manifest = await readJson(manifestPath)
```

Comment:
No error handling? It looks like [`readJSON`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson.md) also allows you to pass it `{ throws: false }` so it doesn't throw if it fails to read the JSON.